src/main/java/lotto/domain/WinningRank.java

```diff
@@ -8,38 +8,49 @@
 public enum WinningRank {
 
     FIRST(6, 2_000_000_000),
-    SECOND(5, 1_500_000),
-    THIRD(4, 50_000),
-    FOURTH(3, 5_000),
+
+    SECOND(5, 30_000_000),
+    THIRD(5, 1_500_000),
+    FOURTH(4, 50_000),
+    FIFTH(3, 5_000),
     NONE(0, 0);
 
-    private final int matchCount;
-    private final int prizeMoney;
+    private final int countOfMatch;
+    private final int moneyOfPrize;
 
-    WinningRank(int matchCount, int prizeMoney) {
-        this.matchCount = matchCount;
-        this.prizeMoney = prizeMoney;
+    WinningRank(int countOfMatch, int moneyOfPrize) {
+        this.countOfMatch = countOfMatch;
+        this.moneyOfPrize = moneyOfPrize;
     }
 
-    public int getMatchCount() {
-        return matchCount;
+    public int getCountOfMatch() {
+        return countOfMatch;
     }
 
-    public int getPrizeMoney() {
-        return prizeMoney;
+    public int getMoneyOfPrize() {
+        return moneyOfPrize;
     }
 
     public static WinningRank valueOf(int matchCount) {
+        return valueOf(matchCount, false);
+    }
+    public static WinningRank valueOf(int matchCount, Boolean isBonusNumberMatching ) {
+
```

Comment:
- 여기는 메서드 구분을 위해 개행을 추가해보면 좋을 것 같습니다.
- primitive 타입을 사용해도 충분할 것 같습니다.
```suggestion
    
    public static WinningRank valueOf(int matchCount, boolean isBonusNumberMatching) {
```